docs/framework/wcf/how-to-host-and-run-a-basic-wcf-service.md

```diff
@@ -1,425 +1,433 @@
-  
-    5.  Step 5 – Open the <xref:System.ServiceModel.ServiceHost> to listen for incoming messages. Notice the code waits for the user to hit enter. If you do not do this, the app will close immediately and the service will shut down.Also notice a  try/catch block used. After the <xref:System.ServiceModel.ServiceHost> has been instantiated, all other code is placed in a try/catch block. For more information about safely catching exceptions thrown by <xref:System.ServiceModel.ServiceHost>, see [Avoiding Problems with the Using Statement](../../../docs/framework/wcf/samples/avoiding-problems-with-the-using-statement.md)  
-  
+
+This is the third of six tasks required to create a Windows Communication Foundation (WCF) application. For an overview of all six of the tasks, see the [Getting Started Tutorial](../../../docs/framework/wcf/getting-started-tutorial.md) topic.
+
+This topic describes how to host a Windows Communication Foundation (WCF) service in a console application. This procedure consists of the following steps:
+
+- Create a console application project to host the service.
+
+- Create a service host for the service.
+
+- Enable metadata exchange.
+
+- Open the service host.
+
+A complete listing of the code written in this task is provided in the example following the procedure.
+
+## Create a new console application to host the service
+
+1. Create a new Console Application project in Visual Studio by right-clicking on the Getting Started solution and selecting **Add** > **New Project**. In the **Add New Project** dialog, on the left-hand side, select the **Windows Desktop** category under **Visual C#** or **Visual Basic**. Select the **Console App (.NET Framework)** template, and then name the project **GettingStartedHost**.
+
+2. Add a reference to the GettingStartedLib project to the GettingStartedHost project. Right-click on the **References** folder under the GettingStartedHost project in **Solution Explorer**, and then select **Add Reference**. In the **Add Reference** dialog, select **Solution** on the left-hand side of the dialog, select GettingStartedLib in the center section of the dialog, and then choose **Add**. This makes the types defined in GettingStartedLib available to the GettingStartedHost project.
+
+3. Add a reference to System.ServiceModel to the GettingStartedHost project. Right-click the **References** folder under the GettingStartedHost project in **Solution Explorer** and select **Add Reference**. In the **Add Reference** dialog, select **Framework** on the left-hand side of the dialog under **Assemblies**. Find and select **System.ServiceModel**, and then choose **OK**. Save the solution by selecting **File** > **Save All**.
+
+## Host the service
+
+Open the Program.cs or Module.vb file and enter the following code:
+
+```csharp
+// program.cs
```

Comment:
Here again, it's useful to pare down the namespaces to the ones actually used (the same ones as in the VB example, along with System. And the filename can be removed.